src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/controller/VisitControllerLegacy.kt

```diff
@@ -73,40 +73,40 @@ class VisitControllerLegacy(
     ) prisonerId: String?,
     @RequestParam(value = "prisonId", required = false)
     @Parameter(
```

Comment:
@tpmcgowan this is the old endpoint you were using, the new one or not so new was being hidden by swagger because it had a defect in it (swagger).